pegasus/sites.v3/code.org/public/educate/it.md

```diff
@@ -14,17 +14,57 @@ theme: responsive
 - All other content, such as K-5 courses (Computer Science Fundamentals) and Hour of Code activities are compatible on iPads and Chromebooks in addition to all other modern tablets and laptop or desktop computers.
 - Looking for mobile device support details? Read below.
 
-## Supported Browsers and Platforms
+## Minimum Supported Browsers and Platforms
+
+<table>
+ <tr>
+  <td style="vertical-align: top; border-color: transparent; padding: 0px 12px 0px 0px;">
+      <table>
+        <th>
+          Browser
+        </th>
+        <tr>
+          <td>Chrome 87.x</td>
+        </tr>
+        <tr>
+          <td>Safari 13.x</td>
+        </tr>
+        <tr>
+          <td>Mobile Safari 11.x</td>
+        </tr>
+        <tr>
+          <td>Edge 87.x</td>
+        </tr>
+        <tr>
+          <td>Firefox 91.x</td>
+        </tr>
```

Comment:
Should we keep this part from being translatable by moving it to a `.partial`? Or using a `.erb` to dynamically populate the table?

Reply:
The plan at the moment is to hide these values in Crowdin so they aren't translated. Is there an advantage to using an `.erb`? I wasn't aware we have any existing dynamic values anywhere in regards to our minimum requirements

Reply:
Oh I forgot that was the plan!